### ISSUE TITLE
Please help with improving accessibility for screen readers.

### ISSUE THREAD

**Issue by Tetrismonster** [CleverRaven/Cataclysm-DDA#55436]
### Is your feature request related to a problem? Please describe.

Hey guys.
I am from the audiogames.net forum.
We are a group of blind and visually impaired gamers who strive to improve accessibility in video games.
We discovered this game some years ago, however, it isn't accessible enough for us to pick up and play.
We use something called a screen reader, which is a piece of software that communicates the contents of the screen via either speech or braille output.
The most widely used screen reader is NVDA, which is free.
We only use the keyboard to play games, not the mouse.
Unfortunately, in order for the game to be made completely accessible, there need to be a few fixes to accessibility.
The windows version is only tile based, and the only accessible text-based version is on linux.
We don't mind installing a vm to play, but it would be great if the windows version could be made accessible.
If making it accessible with a screen reader isn't possible, you could use microsofts speech API.
However, please if you decide to do this could you give us options to customize the voice we use? Changing rate, pitch, voice etc.
Some have more than one voice on their machine, and this would help make the game as playable as possible.
Another thing that we need is an option to completely disable ascii, as that will make the screen reader say things like slash slash slash slash slash slash backslash backslash and so on.
If the windows version cannot be made accessible, could you please fix the linux curses version?
Right now it is sort of playable, but many members of our community, including me, are too scared to mess with it in its current state.
Linux uses a screen reader called orca, and the game cannot be played with anything before ubuntu 18.
Apparently the game is completely inaccessible when trying to play with later versions.
The menu options don't automatically read when we press the arrow keys. Also, would it be possible to have the speech read the info as it gets displayed? You could try playing a mud using a mud client like vip mud which you can get from gmagames.com for reference.
There is a trial version, don't worry.
Just make sure you have nvda installed.
We would also need text descriptions, as well as the ability to understand what's around us.
Also, if you decide to make the windows version accessible, soundpacks can also help by providing bits of info with different audio cues.
Let me know if you have any questions.
Thanks for reading.

### Solution you would like.

If the windows version isn't able to be made accessible with a screen reader, microsoft sapi could be used.
The game would need to have all output read by the screen reader when it appears.
The menus need to be made accessible for screen readers, or we could use the numbers row to choose options.
Instead of ascii, short text descriptions will suffice.

**Comment by storyofablindlife** [comment]
Having this game accessible on Windows would be great, I totally agree. Games like Stardew Valley are now accessible, with a keypress we can know what's on the tile we're facing, be it grass or an object, and NVDA reads it. Maybe something similar could be added to CDDA.

**Comment by Zireael07** [comment]
Accessibility issues should be excluded from stalebot!

**Comment by Zireael07** [comment]
Bot autoclosed a key accessibility issue!
